### ISSUE TITLE
[Feature Request]: Enable import of existing isa.assay.xlsx tables

### ISSUE THREAD

**Issue by micwij** [nfdi4plants/ARCtrl#414]
### Contact Details

micwij@gmail.com

### Is your feature request related to a problem?

I am currently a bit stuck with an old ARC that was created some time ago with the ARCCommander, which does not open in the ARCitect. So to edit annotations I can neither use the Swate from within ARCitect nor, the Swate excel addin because these are managed by my institution.

### Describe the solution you would like!

For this very specific case it would be great to be able to re-import existing isa.assay.xlsx tables and I think this might generally be useful.

### Additional context!

See: https://github.com/nfdi4plants/ARCCommander/issues/240

### Host

Standalone

### Host Details

Swate: Standalone
Browser: Microsoft Edge

**Comment by Freymaurer** [comment]
@HLWeil i think it would be best to have a function, which verifies if a worksheet name belongs to assay/study/investigation, which checks for current as well as obsolete names.

**Comment by HLWeil** [comment]
Should be easy to implement. Something in the lines of:

`ArcAssay.isMetadataSheetName: string -> bool`

and

`ArcAssay.isMetadataSheet: FsWorksheet -> bool`

?

**Comment by Freymaurer** [comment]
Yes! Or even: `ArcAssay.tryMetadataSheetName (fswb: FsWorkbook)`
